tests/scenario/modules/models/asset-model-metadata-propagation.test.ts

```diff
@@ -0,0 +1,361 @@
+import { ResponsePayload } from "kuzzle-sdk";
+import { PartialDeep } from "type-fest";
+import {
+  ApiAssetCreateRequest,
+  ApiAssetGetRequest,
+  ApiAssetGetResult,
+} from "../../../../lib/modules/asset";
+import {
+  ApiModelDeleteAssetRequest,
+  ApiModelWriteAssetRequest,
+} from "../../../../lib/modules/model";
+import { setupHooks } from "../../../helpers";
+
+jest.setTimeout(10000);
+
+describe("Asset model metadata propagation", () => {
+  const sdk = setupHooks();
+
+  beforeAll(async () => {
+    if (
+      await sdk.document.exists(
+        "device-manager",
+        "models",
+        "model-asset-Pallet"
+      )
+    ) {
+      await sdk.query<ApiModelDeleteAssetRequest>({
+        controller: "device-manager/models",
+        action: "deleteAsset",
+        _id: "model-asset-Pallet",
+      });
+    }
+  });
+
+  afterEach(async () => {
+    await sdk.query<ApiModelDeleteAssetRequest>({
+      controller: "device-manager/models",
+      action: "deleteAsset",
+      _id: "model-asset-Pallet",
+    });
+  });
+
+  it("should add new metadata with default value set on linked assets in all engines", async () => {
+    await sdk.query<ApiModelWriteAssetRequest>({
+      controller: "device-manager/models",
+      action: "writeAsset",
+      body: {
+        engineGroup: "commons",
+        model: "Pallet",
+        metadataMappings: {
+          depth: { type: "integer" },
+          width: { type: "integer" },
+        },
+        defaultValues: {
+          depth: 80,
+          width: 120,
+        },
+      },
+    });
+
+    await sdk.query<ApiAssetCreateRequest>({
```

Comment:
put assets creation in fixtures file ?

Reply:
There's some variations in the assets between tests